core/domain/acl_decorators.py

```diff
@@ -68,6 +76,39 @@ def check_activity_accessible(
             activity_rights.viewable_if_private)
 
 
+def check_exploration_editable(user_id, user_actions, exploration_rights):
+    """Returns a boolean to signify whether given exploration is editable
+    by the user or not.
+
+    Args:
+        user_id: str. Id of the given user.
+        user_actions: list(str). List of actions given user can perform.
+        exploration_rights: Exploration rights object.
```

Comment:
Existence of exploration is not checked in can_view_exploration_feedback decorator. I did this as there was no check for such case before, so I maintained current behaviour.
But on testing a bit I realize that it shows server error on inexistent exploration id, so a check should be placed here as well.

Reason for passing exploration_rights instead of exploration_id:
We decided to pass _id_ and _actions_ as separate arguments instead of _self_ object. But, _PageNotFoundException_ belongs to _self_. So, I passed exploration_rights instead in order to avoid passing the exception.
(But I also think passing exp_id makes more sense.)
Now, should I pass another argument page_not_found_exception (which doesn't seem right as it is always same, self.PageNotFoundException) or should I shift to passing the _self_ object?